ctms/crud.py

```diff
@@ -477,11 +477,12 @@ def create_waitlist(
 ) -> Optional[Waitlist]:
     if waitlist.is_default():
         return None
-    if not isinstance(waitlist, WaitlistInSchema):
-        # Sample data are used as both input (`WaitlistInSchema`) and internal (`WaitlistSchema`)
-        # representations.
-        waitlist = WaitlistInSchema(**waitlist.dict())
-    db_waitlist = Waitlist(email_id=email_id, **waitlist.orm_dict())
+
+    attrs = waitlist.dict()
+    if "subscribed" in attrs:
+        del attrs["subscribed"]
```

Comment:
Just want to make sure I understand what's happening here:

- `create_waitlist` is used in `create_contact` and `create_or_update_contact`
- It uses the `WaitlistInSchema`, which has a `subscribed` property 
  - this makes sense because we're receiving this payload from `Basket`, and waitlists are just newsletters with naming conventions, so it has this property
- we want to use this property to determine whether or not we should delete or create the waitlist for the contact, but our `Waitlist` ORM model currently doesn't have an update property, so we need to remove it

Reply:
Yes, this is it.
I'll copy part of your note in a code comment